create_dmd_release/create_dmd_release.d

```diff
@@ -602,6 +607,8 @@ void createRelease(string branch)
         {
             copyFile(cloneDir~"/druntime/lib/gcstub.obj", osDir~"/lib/gcstub.obj");
             copyFile(cloneDir~"/phobos/phobos.lib", osDir~"/lib/phobos.lib");
+            foreach (de; dirEntries(cloneDir~"/druntime/lib/win32/", "*.lib", SpanMode.shallow))
```

Comment:
You can use `copyDir(cloneDir~"/druntime/lib/win32/", osDir~"/lib/", file => file.endsWith(".lib"))` instead.